features/machine/autoscaler.feature

```diff
@@ -174,7 +174,7 @@ Feature: Cluster Autoscaler Tests
       | resource | machineset                         |
       | name     | <%= cb.machineset_clone_22102_2 %> |
     Then the step should succeed
-    And the output should match "Annotations:\s+<none>"
+    And the output should not match "autoscaling.openshift.io/machineautoscaler"
```

Comment:
Same here, the test run `describe` 3 times, which is time consuming, run it once and use the `match` clause 3 times.